com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -853,6 +860,15 @@ private void NetworkUpdate()
                     currentNetworkTimeOffset += Mathf.Clamp(networkTimeOffset - currentNetworkTimeOffset, -maxDelta, maxDelta);
                 }
             }
+
+            var profileTransport = NetworkConfig.NetworkTransport as ITransportProfilerData;
```

Comment:
This code is totally fine but I always like to share newer c# syntax, this could also be
`if(NetworkConfig.NetworkTransport is ITransportProfilerData profileTransport)`

(tbh I'm still not 100% sold on the concept of inline variable declarations, it's been growing on me)